app/models/concerns/twitter_user/utils.rb

```diff
@@ -0,0 +1,53 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Utils
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def has_more_than_two_records?(uid, user_id)
+      where(uid: uid, user_id: user_id).limit(2).pluck(:id).size >= 2
+    end
+  end
+
+  included do
+    def self.oldest(user, user_id)
+      user.kind_of?(Integer) ?
+        order(created_at: :asc).find_by(uid: user.to_i, user_id: user_id) : order(created_at: :asc).find_by(screen_name: user.to_s, user_id: user_id)
+    end
+
+    def self.latest(user, user_id)
+      user.kind_of?(Integer) ?
+        order(created_at: :desc).find_by(uid: user.to_i, user_id: user_id) : order(created_at: :desc).find_by(screen_name: user.to_s, user_id: user_id)
```

Comment:
__[rubocop]__
* Line is too long. [151/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)